circle.yml

```diff
@@ -15,4 +15,3 @@ jobs:
             - node_modules
       - run: npm test
       - run: npm run lint
-      - run: npm run security
```

Comment:
so sad npm depricated this

Reply:
I wonder if @fuchaimead would be willing to remove the script from the package.json file?